src/clients/HttpTunneler.cc

```diff
@@ -278,6 +278,12 @@ Http::Tunneler::handleResponse(const bool eof)
         hp = new Http1::ResponseParser;
 
     auto parsedOk = hp->parse(readBuf); // may be refined below
+
+    debugs(11, 2, "Tunnel Server " << connection);
+    debugs(11, 2, "Tunnel Server RESPONSE:\n---------\n" <<
+           Raw(nullptr, readBuf.rawContent(), readBuf.length()).minLevel(2).gap(false) <<
```

Comment:
AFAICT, this location may print multiple RESPONSE messages per response and this code may print a lot more than the response header even when parsedOk is true. Can we do something similar to what HttpStateData::processReplyHeader() does instead?

Reply:
Adjusted (6f56194).